server/Quiz.ts

```diff
@@ -0,0 +1,53 @@
+import EventEmitter from "https://deno.land/x/events/mod.ts";
+import { v4 } from "https://deno.land/std/uuid/mod.ts";
+import Student from "./Student.ts";
+
+class Quiz extends EventEmitter {
+    id_from_lecturer: String;
+    server_id: String;
+    time_seconds: number;
+    questions: any;
+    student_ids: String[];
+    student_answers: Map<String, any>;
+    active: boolean;
+
+    constructor(id_from_lecturer: String, time_seconds: number, questions: any, student_ids: String[]) {
+        super();
+        this.id_from_lecturer = id_from_lecturer;
+        this.time_seconds = time_seconds;
+        this.server_id = v4.generate();

```

Comment:
`this.server_id = v4.generate()` this looks more like quiz id not server id.

Reply:
yeah, i forgot to change it to something like id_from_server